core/src/components/badge/badge.ionic.scss

```diff
@@ -174,3 +175,23 @@
   min-width: globals.$ion-scale-400;
   height: globals.$ion-scale-400;
 }
+
+// Badge Inside Tab Button
+// --------------------------------------------------
+
+:host([vertical].in-tab-button) {
+  position: relative;
+
+  min-width: $badge-ionic-in-tab-button-min-width;
+}
+
+:host([vertical].in-tab-button) ::slotted(ion-icon) {
+  @include globals.position($top: 50%, $start: 50%);
+
+  position: absolute;
+
+  width: $badge-ionic-in-tab-button-icon-size;
+  height: $badge-ionic-in-tab-button-icon-size;
+
+  transform: translate(-50%, -50%);
+}
```

Comment:
It seems that this code is not needed if we adjust the styles as shown in common. This should be verified.